SandB-iOS/SandB-iOS/Controllers/ArticlesListTableViewController.m

```diff
@@ -56,35 +43,34 @@ - (void)viewDidLoad
         //NSLog(@"Received Category: %@", self.recievedCategory);
     }
     
-    
     if ([self.newsCategory isEqualToString:@"News"]) {
         
         //Check the cache content.
         NSMutableArray *news =  [[Cache sharedCacheModel] loadArchivedObjectWithFileName:@"News"];
+        
         if (news) {
-            //NSLog(@"loading articles from cache");
-            
             self.allArticlesArray = news;
         }
+        
         [self fetchArticles];
-    }
-    else if ([self.newsCategory isEqualToString:@"Favorites"]) {
+        [self.refreshControl beginRefreshing];
+        
+    } else if ([self.newsCategory isEqualToString:@"Favorites"]) {
         // Do nothing on favorites.
         self.allArticlesArray =  [[DataModel sharedModel] savedArticles];
         [self.tableView reloadData];
-    }
-    else {
         
+    } else {
         //Check cache content of category.
         NSMutableArray *categoryCache = [[Cache sharedCacheModel] loadArchivedObjectWithFileName:self.recievedCategory];
```

Comment:
Same thing but this block uses fetchCategoryArticles and the first uses fetchArticles. Same thing happens all over this file. This all goes back to two different methods in DataModel. Going forward I would want to move that distinction behind a fetchArticlesForCategory method in the data model that does something special (`get_recent_posts`) for the "News" category, and this VC should always call the same method. But for now I'll refactor this a little to make it less repetitive.